### DISCUSSION TITLE
First day of the week in calendar widget

### DISCUSSION THREAD

**Discussion by CrumblyLiquid** [elkowar/eww#887] [General]
Does the calendar widget have a way to set the first day of the week to be monday?

The [widget documentation](https://elkowar.github.io/eww/widgets.html) doesn't say anything about it but I'm hoping it's just not documented.

**Comment by mantacid** [comment]
Intuition tells me that it's based on the gtk calendar widget, so maybe check their docs to see if theirs an environment variable you can set.
